### ISSUE TITLE
React.createClass is deprecated

### ISSUE THREAD

**Issue by hwillson** [meteor/react-packages#219]
Migrating this over from meteor/meteor#8625. `React.createClass` is deprecated, so the `react-meteor-data` package should be updated accordingly (PR #218 has been created to help with this).

**Comment by zeroasterisk** [comment]
I tried to forcefully set the new version and it was not found :(

```

=> Started proxy.
=> Started MongoDB.
=> Errors prevented startup:

   While selecting package versions:
   error: No version of react-meteor-data satisfies all constraints: @0.2.10
   Constraints on package "react-meteor-data":
   * react-meteor-data@0.2.10 <- top level
```

**Comment by hwillson** [comment]
Hi @zeroasterisk - version `0.2.10` is an older version that wasn't published. A newer version is coming shortly (should be published today) and will include a `React.createClass` deprecation fix. It's taking a bit longer to publish as we're deciding whether to include #217 in the next release (which would essentially nullify the fix for this issue made in #218). Either way, this should be done today. Thanks for your patience!

**Comment by abernix** [comment]
If anyone wants to try this right now, I've published `react-meteor-data@0.2.11-beta.1` which includes #217 on top of #218.

/cc @merlinpatt @leocavalcante

**Comment by lmachens** [comment]
@abernix beta is working fine for me. 
When will you publish the final version?

**Comment by abernix** [comment]
Thanks for the reminder, @lmachens!

I was waiting for a few 👍 s.  It has been published as `react-meteor-data@0.2.11`.

**Comment by hwillson** [comment]
Since the new `react-meteor-data` version has been published, this issue can now be closed. Thanks all!
